### DISCUSSION TITLE
https://widcard.win/posts/blog/blog-transfer2

### DISCUSSION THREAD

**Discussion by giscus** [widcardw/giscus-discussion#9] [Announcements]
# https://widcard.win/posts/blog/blog-transfer2

Opinionated Vite Starter Template

https://widcard.win/posts/blog/blog-transfer2

**Comment by widcardw** [comment]
所以大家还是拿现成的框架写博客，不要像我一样搁这捣鼓半天 ¯\\\_(ツ)\_/¯
